adobe/react-spectrum
# Fixes #2248: Updating plop templates to generate components with dynamic filenames

Closes #2248 

Current workflow: 
- plop generates components 
- plop action `renameMany.js` is used to rename the generated files, to include the component name 

What this PR brings:
- Plop now puts the component name directly in the templates. Hence, now the extra step of using plop-action renameMany becomes obsolete.

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

- Run `yarn run plop` and answer prompts
- Observe the generated files


## 🧢 Your Project:
